deployment/helm/node-feature-discovery/templates/cert-manager-certs.yaml

```
@@ -0,0 +1,66 @@
+{{- if .Values.tls.certManager }}
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: nfd-master-cert
+spec:
+  secretName: nfd-master-cert
+  subject:
+    organizations:
+    - node-feature-discovery
+  commonName: nfd-master
+  dnsNames:
+  # must match the service name
+  - {{ include "node-feature-discovery.fullname" . }}-master
+  # first one is configured for use by the worker; below are for completeness  
+  - {{ include "node-feature-discovery.fullname" . }}-master.{{ $.Release.Namespace }}.svc
+  - {{ include "node-feature-discovery.fullname" . }}-master.{{ $.Release.Namespace }}.svc.cluster.local
+  # localhost needed for grpc_health_probe
+  - localhost
+  issuerRef:
+    name: nfd-ca-issuer
+    kind: Issuer
+    group: cert-manager.io
+
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: nfd-worker-cert
+  namespace: node-feature-discovery
```

I think you need to use `{{ $.Release.Namespace }}` here, too. Ditto all the other yamls, too...

```suggestion
  namespace: {{ $.Release.Namespace }}
```

---

I think I actually need to remove it entirely (which I've done); helm itself will manage the namespace it's being installed into.

---

Uh-oh, true 👍 Sorry for my helm-noobness 🙄